rules/windows/process_creation/proc_creation_win_schtasks_curl_and_powershell_combo.yml

```diff
@@ -0,0 +1,36 @@
+title: Schtasks Curl Download and Powershell Execution Combination
```

Comment:
I'm stuck on it but I have no (good) idea
Scheduled Task Creation With wget and powershell Via Schtasks.EXE
suspicious Schtasks commandline with  Curl Download and Powershell Execution Combination